test/integration/nats_jetstream/nats_test.go

```diff
@@ -0,0 +1,147 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package nats_jetstream
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/nats-io/nats.go"
+
+	ce_nats "github.com/cloudevents/sdk-go/protocol/nats_jetstream/v2"
+	"github.com/cloudevents/sdk-go/v2/binding"
+	"github.com/cloudevents/sdk-go/v2/event"
+	bindings "github.com/cloudevents/sdk-go/v2/protocol"
+	"github.com/cloudevents/sdk-go/v2/protocol/test"
+	. "github.com/cloudevents/sdk-go/v2/test"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	. "github.com/cloudevents/sdk-go/v2/binding/test"
+)
+
+func TestSendReceiveStructuredAndBinary(t *testing.T) {
+	t.Skip("Need a running NATS server with the -js option.  Turning on -js will break non jetstream tests.")
```

Comment:
The current use of `nats-streaming` implementation in our CI is deprecated. In order to not push too much work on your end here, please update the Github integration [workflow](https://github.com/cloudevents/sdk-go/blob/754a26bb34647e89a32228d136a86e687c86ee8a/.github/workflows/integration.yaml#L21) by adding a separate service for `nats-jetstream` and configure your tests to leverage this instance.

Skipping tests should not be an option here :)

Reply:
@embano1 , changed the port to 4223.  re-enabled the test.  
Running docker locally with "docker run -p 4223:4223 nats:latest -js -p 4223"
Let me know if my github actions workflow looks correct.